.flowconfig

```diff
@@ -11,18 +11,26 @@
 ; Ignore duplicate module providers
```

Comment:
Can you add a link where you got the changes from or did you manually change it? :)

Reply:
https://react-native-community.github.io/upgrade-helper/?from=0.59.9&to=0.60.6